dapps/marketplace/src/pages/growth/Invite.js

```diff
@@ -52,15 +52,13 @@ class GrowthInvite extends Component {
   }
 
   getInviteCode() {
-    return `${location.protocol}//${location.hostname}/#/welcome/${this.state.inviteCode}`
+    return `https://originprotocol.com/referral/${this.state.inviteCode}`
```

Comment:
We have 2 types of invite codes:
 a. code generated by users. Those point to https://shoporigin.com/#/welcome/<userCode>
 b. code generated for partners. Those point to https://originprotocol.com/referral/<partnerCode>
Won't this change break a. ?

CC @mikeshultz

Reply:
No, it shouldn't break anything.  It just isn't as direct as before.  The originprotocol.com landing page will provide them with a mobile app or desktop link to the URL in (a).